packages/@react-aria/i18n/src/useNumberParser.ts

```diff
@@ -10,39 +10,243 @@
  * governing permissions and limitations under the License.
  */
 
-import {useCallback, useEffect, useRef} from 'react';
+import {useCallback, useMemo} from 'react';
 import {useLocale} from './context';
+import {useNumberFormatter} from './useNumberFormatter';
 
 type NumberParser = {
-  parse: (value:string) => number
+  parse: (value: string) => number,
+  round: (value: number) => number,
+  symbols: {
```

Comment:
I'd like to avoid exposing these outside the number parser if we can. Looks like that might be possible if we combine the `cleanString` function with the rest of the parser.

Reply:
almost, still need to leave the signs in so we can handle them in the component since the parser doesn't know about min/max values